layers/meta-balena-beaglebone/recipes-support/resin-init/resin-init-flasher.bbappend

```diff
@@ -3,9 +3,14 @@ INTERNAL_DEVICE_KERNEL:beaglebone = "mmcblk1"
 # Except on the beaglebone-ai64
 INTERNAL_DEVICE_KERNEL:beaglebone-ai64 = "mmcblk0"
 
+# Because eMMC enumeration on the beagpleplay is different from other Beaglebone devices (?)
+INTERNAL_DEVICE_KERNEL:beagleplay = "mmcblk0"
+
 # the name of the bootloader config file to be written to internal media
 INTERNAL_DEVICE_BOOTLOADER_CONFIG:beaglebone = "uEnv.txt_internal"
+INTERNAL_DEVICE_BOOTLOADER_CONFIG:beagleplay = "uEnv.txt_internal"
```

Comment:
I don't think we use this so it can be removed

Reply:
I have made this change.